openwrt/packages
# unbound: improve startup and dhcp script robustness

Maintainer: me
Tested: Linksys WRT3200ACM
Description:
- prevent rapid overlap in DHCP script updates
- check and allow localhost forwards with specific applications
- add option for rate limiting inbound queries
- change UCI list to table format with Unbound conf references

enhances: #12581
resolves: OpenWrt [Forum 66354](https://forum.openwrt.org/t/issues-with-unbound-and-odhcp-setup/66354/19)
